src/libstd/macros.rs

```diff
@@ -24,6 +24,8 @@
 /// The multi-argument form of this macro panics with a string and has the
 /// `format!` syntax for building a string.
 ///
+/// If the main thread panics it will return with code `101`.
```

Comment:
it's currently hard-coded; I don't know if we want to commit to this

@rust-lang/lang